### ISSUE TITLE
moderators widget

### ISSUE THREAD

**Issue by pdpinch** [mitodl/open-discussions#1053]
Display a simple widget in the right rail with a list of moderators, including their profile image and headline. 

<img width="281" alt="screenshot 2018-08-13 14 53 37" src="https://user-images.githubusercontent.com/430126/44051731-adc4ac86-9f08-11e8-893d-b3353b327e92.png">

### out of scope:
- #1062 modifications to moderators API to support this widget
- Later, we will have a widget system, where channel owners can chose whether to display this widget on their channels.

**Comment by pdpinch** [comment]
Blocked by design questions, like what to do if there are many moderators.

**Comment by pdpinch** [comment]
Updated with link to new, separate issue for back-end API work.

**Comment by pdpinch** [comment]
This was addressed by #1803
